### PR TITLE
127 add timestamped transcription workflow

### LINKED ISSUE #127: Pipeline: creating a pipeline combining speech to text + forced alignment
hi @ibevers , I have a new task for you and the reason I ask you is that you already have experience with forced alignment.
Can you please draft a function with the following characteristics?
INPUTS:
- audios (list of Audio objects)
- speech to text model (a huggingface model, you can make whisper as the default)
- language (language object, optional - you can make None the default - for whisper, this will trigger language detection)

OUTPUT:
- list of scriptline objects resulting from the transcript PLUS forced alignment

internally, you should do all the preprocessing (downmixing channels, resampling), speech to text and forced alignment. 

We want this pipeline to become our robust pipeline for doing timestamped transcriptions.

### PR DESCRIPTION
**Pull request by ibevers**
- [x] Tutorial
- [x] Tests
- [x] Update __init__.py

### CODE REVIEW

**Comment by ibevers** [comment]
@fabiocat93 do you know why that one test case might be failing?

**Comment by fabiocat93** [comment]
> @fabiocat93 do you know why that one test case might be failing?

that's because you have commented the `**kwargs` param in [src/senselab/audio/tasks/speech_to_text/api.py](https://github.com/sensein/senselab/pull/138/files#diff-39542f816ec55fd4355a7ad73e8848c10d7fbf3405db483ee0e64c70aa9c2ca9). can i ask you why? that was for passing model specific params to the pipeline. if you make it back how it was, it should work

**Review by fabiocat93 — CHANGES_REQUESTED** [review]
please review the `**kwargs` param in [src/senselab/audio/tasks/speech_to_text/api.py](https://github.com/sensein/senselab/pull/138/files#diff-39542f816ec55fd4355a7ad73e8848c10d7fbf3405db483ee0e64c70aa9c2ca9)

**Comment by ibevers** [comment]
@fabiocat93 ah, right, I forgot I did that. Taking a look...

File: src/senselab/audio/tasks/speech_to_text/api.py [hunk 1]
```diff
@@ -40,7 +40,10 @@ def transcribe_audios(
     try:
         if isinstance(model, HFModel):
             return HuggingFaceASR.transcribe_audios_with_transformers(
-                audios=audios, model=model, language=language, device=device, **kwargs
+                audios=audios,
+                model=model,
+                language=language,
+                device=device,  # , **kwargs
```
Comment (fabiocat93) [inline src/senselab/audio/tasks/speech_to_text/api.py]: i think this should also be uncommented @ibevers

**Comment by fabiocat93** [comment]
> @fabiocat93 ah, right, I forgot I did that. Taking a look...

no problem, just let me know when you are done

**Comment by ibevers** [comment]
@fabiocat93 done

**Comment by fabiocat93** [comment]
> @fabiocat93 done

Great!! thanks!

**Review by fabiocat93 — APPROVED** [review]

**Pull request merged**